lite-variables.tf

```diff
@@ -185,6 +178,16 @@ variable "thor_num_workers" {
   default    = 2
 }
 
+variable "thor_worker_cpus" {
```

Comment:
This is not something the user should be required to set.  It should be derived from the VM size you choose for the thorpool.

Reply:
Knowing the VM size, which means you know the number cpus on each node, isn't enough to determine 'thor_worker_cpus' because you can have more than one thor worker on a node.  I do have a default value for 'thor_worker_cpus' of 2. I also, restrict 'thor_worker_cpus' to 2, 4, 8, 16 because I want it to be a factor of the number of cpus on each node.

Reply:
1. Also, I now calculate 'thor_worker_ram'.

Reply:
Again, this is not something the user should be required to set.

Let's simplify this for now:  Thor workers will always use 2 vCPUs.  Please remove this TF variable and use a constant of 2 cores for future computations.

Reply:
fixed